Javascript/Code-Challenges/linked-list/README.md

```diff
@@ -0,0 +1,29 @@
+# Collection: Linked List
+
```

Comment:
For readme it one per challenge for example readme-06.md(or whatever you want to call it) that way it easier to find the information instead of over writing, or scrolling all the way down each time you do a new challenge. For each challenge it should have in the readme
```
# Singly Linked List
<!-- Short summary or background information -->

## Challenge
<!-- Description of the challenge -->

## Approach & Efficiency
<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->

## API
<!-- Description of each method publicly available to your Linked List -->
```